src/shared/components/palette/case-flag/components/manage-case-flags/manage-case-flags.component.ts

```diff
@@ -83,8 +87,8 @@ export class ManageCaseFlagsComponent implements OnInit {
       : '';
 
     return flagPathOrName === flagOtherDescriptionOrName
-      ? `${partyName} - ${flagOtherDescriptionOrName}${comment}`
-      : `${partyName} - ${flagPathOrName}, ${flagOtherDescriptionOrName}${comment}`;
+      ? `${partyName}${flagOtherDescriptionOrName}${comment}`
```

Comment:
@johnbenjamin-hmcts - just checking, is the spacing definitely correct? The flag description or flag path/name appears to be right next to the party name

Reply:
@Daniel-Lam - The spacing is handled in the same method when figuring out `partyName` and `comment`.